Source/Private/ASDisplayNodeInternal.h

```diff
@@ -127,6 +127,9 @@ FOUNDATION_EXPORT NSString * const ASRenderingEngineDidDisplayNodesScheduledBefo
   ASDisplayNode * __weak _supernode;
   NSMutableArray<ASDisplayNode *> *_subnodes;
 
+  // Set this to nil whenever you modify _subnodes
+  NSArray<ASDisplayNode *> *_cachedSubnodes;
```

Comment:
This seems like a dangerous optimization. Could we at least put in an assert that _subnodes and _cachedSubnodes have the same content in the subsides getter?